cmd/src/campaign_progress_printer.go

```diff
@@ -55,7 +55,7 @@ func (p *campaignProgressPrinter) initProgressBar(statuses []*campaigns.TaskStat
 	}
 
 	p.progress = p.out.ProgressWithStatusBars([]output.ProgressBar{{
-		Label: fmt.Sprintf("Executing steps in %d repositories", len(statuses)),
+		Label: fmt.Sprintf("Executing ... (  0/%d,   0 errored)", len(statuses)),
```

Comment:
Just to avoid an initial bounce as the first 9 changesets are executed (since we use `%d` below without a width specifier), this might be better without the leading spaces:

```suggestion
		Label: fmt.Sprintf("Executing ... (0/%d, 0 errored)", len(statuses)),
```

Reply:
Hehe, this is why it's a 🙈 draft 🙈 What I was trying to do here is to add enough buffer whitespace to the `Label` so that it's not cut off later once the the numbers get bigger.

I might also add a "SetLabelAndRecalc" method to the progress bar.